.gitignore

```diff
@@ -6,3 +6,5 @@ spack-build*
 /src/docs/doxygen/html/
 /src/docs/doxygen/xml/
 /src/docs/sphinx/_build/
+*.cmake
+.vscode/
```

Comment:
Note: some projects (MFEM 😛) don’t like to use the local .gitignore to ignore IDE config files. The reason is to prevent .gitignore from containing all the possible IDEs config file.

I think it’s not a bad idea, especially since the alternative is quite convenient: 
You can ignore those file globally.

Add the vscode pattern in a file named, for example `~/.gitignore_global`
Run `git config --global core.excludesfile ~/.gitignore_global` (one-time command to register you global gitignore file)

Reply:
True. I generally like to have clean git status output and not pollute my global git config since I work on many projects. However, I don't feel that strongly about it so others (@white238 , @samuelpmish , @jonwong12 ) should definitely chime in.

Reply:
It's fine the way it is.